### PR TITLE
Properly populate recent regular and stake transactions

### PR DESCRIPTION
**Pull request by alexlyp**
Also add some various styling cleanup and refinement.

### CODE REVIEW

**Comment by vctt94** [comment]
Good job, LGTM

The only thing I noticed is when clicking on tickets we get an error message 
`Cannot read property 'rawTx' of undefined`

**Comment by alexlyp** [comment]
Hrm can't seem to replicate this issue.  All you do is click on a ticket listed in the Recent Ticket activity area?  It loads a tx details totally fine for me

File: app/style/HomePage.less [hunk 1]
```diff
@@ -171,9 +177,21 @@
...
 
+.tx-history-overview-row.Vote .icon {
+  background-image: @ticket-voted-icon;
+}
+
+.tx-history-overview-row.Ticket .icon {
+  background-image: @ticket-small-icon;
+}
+
+.tx-history-overview-row.Revoke .icon {
```
Comment (matheusd) [inline app/style/HomePage.less]: this should be `.Revocation`, otherwise the icon is not shown

**Comment by vctt94** [comment]
Yes I just click on a vote actually. Tickets seems to work fine. Here is the log I am getting: 

bug

And the error is at TransactionPage: 

`if (!viewedDecodedTransaction) {
    decodeRawTransactions([viewedTransaction.rawTx]);
  }`

**Comment by vctt94** [comment]
On master I have this: 

bug

I believe that is why the ticket worked. I just bought it, and it loads. If I have made many transactions I believe it would crash too. I will try it.

**Pull request merged**